srcpkgs/bind/template

```diff
@@ -5,14 +5,10 @@ revision=1
 _fullver="${version}${_patchver:+-${_patchver}}"
 wrksrc="${pkgname}-${_fullver}"
 build_style=gnu-configure
-configure_args="--disable-static --enable-threads --enable-largefile
- --with-libtool --enable-atomic --sysconfdir=/etc/named --enable-epoll
- --enable-ipv6 --with-randomdev=/dev/random --with-ecdsa=yes --with-eddsa=no
- --with-libtool --with-openssl=${XBPS_CROSS_BASE}/usr --with-gssapi=/usr/bin
- --without-gost --enable-openssl-hash --with-readline --with-tuning=default
- --without-python --enable-fetchlimit --enable-sit
- --with-libidn2 --disable-backtrace
- $(vopt_if geoip "--with-geoip=${XBPS_CROSS_BASE}/usr" "--without-geoip")"
```

Comment:
Is `geoip` automatically detected now?

Reply:
--disable-geoip         support GeoIP2 geolocation ACLs if available [default=yes]

So yes. That reminds me, I was going to include my findings from the ticket in the commit message.